src/app/containers/Footer/index.test.jsx

```diff
@@ -41,6 +41,9 @@ const contextStub = {
     brandBackgroundColour: `${C_POSTBOX}`,
     brandLogoColour: `${C_WHITE}`,
   },
+  translations: {
```

Comment:
This is something that I hadn't noticed previously - the skip to content link is now in the brand component, the brand component is in the footer as well as the header, so the skip to content link is in the footer. Is this intentional or do we need to change this?

Reply:
We could very easily compose the skip link into the brand container if we don't want this in the footer.

Reply:
I've checked this with laura - we don't want to show the skip to content link in the footer!

Reply:
Fixed this!